### ISSUE TITLE
IPv6 rich rule not interpreting to-addr correctly

### ISSUE THREAD

**Issue by carolyn76** [firewalld/firewalld#422]
I have created an ipv6 rich rule to forward all ipv6 traffic for udp port 162 to a specific address, port 162.  When I attempt to restart the service it doesn't fail, but it will immediately render my vm not accessible via ssh.  When I access the virtual console(vm is in openstack) and run a systemctl status firewalld, it is complaining with 'Bad IP address "x:x:x:x:x:x:x:x:162".  I attempted to add the rule with [ ] around the ipv6 address, but it will not take that.  Is there something I am missing for ipv6?

Command I am running to add the rule:

firewall-cmd --permanent --zone=public --add-rich-rule='rule family="ipv6" source address="::/0" forward-port to-addr="x:x:x:x:x:x:x:x" to-port="162" protocol="udp" port="162"'

Bottom line is it is including the port 162 in the address and not seperating it with [ ].
